src/modules/blockchainApplication/manage/components/NetworkApplicationDropDownButton/NetworkApplicationDropDownButton.js

```diff
@@ -0,0 +1,47 @@
+import React, { useCallback } from 'react';
+import { getLogo } from '@token/fungible/utils/helpers';
+import { TertiaryButton } from 'src/theme/buttons';
+import Icon from 'src/theme/Icon';
+import { addSearchParamsToUrl } from 'src/utils/searchParams';
+import routes from 'src/routes/routes';
+import Network from '@common/components/bars/topBar/networkName';
+import { useCurrentApplication } from '../../hooks/useCurrentApplication';
+import styles from './NetworkApplicationDropDownButton.css';
+
+const ApplicationManagementList = ({ history, location }) => {
```

Comment:
The filename and export component name are different? `NetworkApplicationDropDownButton.js`

Reply:
Yes hate when they are different, fixed here https://github.com/LiskHQ/lisk-desktop/pull/4893/commits/49f528607e5bb5c216eda6f800dbe3e0a848b1cf